client/src/templates/Challenges/components/CompletionModalBody.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
```

Comment:
```suggestion
import React, { PureComponent } from 'react';
```
With this it will only render if the state or the props differ from the previous render.